tools/create_testgrid_tickets.py

```diff
@@ -0,0 +1,161 @@
+
+def create_jira_ticket(jclient, existing_tickets, test_id, test_data):
+    last_pass_timestamp = get_last_pass(test_data)
+    summary = format_summary({"test_id":test_id,
+                              "last_pass": last_pass_timestamp})
+    if summary in existing_tickets:
+        logger.debug("issue found: %s", summary)
+        return None
+
+    failed_tests = [t['display_name'] for t in test_data['tests'] if t['display_name'] != 'Overall']
+    test_status = test_data['status']
+    i = get_last_failure_instance(test_id)
+    new_issue = jclient.create_issue(project="MGMT",
+                                     description=format_description({"test_id": test_id,
+                                                                     "last_fail": i,
+                                                                     "failed_tests": failed_tests,
+                                                                     "status": test_status,
+                                                                    }),
+                                     summary=summary,
+                                     versions=[{'name': 'OpenShift {}'.format(TEST_VERSION_REGEX.findall(test_id)[0])}],
+                                     components=[{'name': "Assisted-installer Triage"}],
+                                     priority={'name': 'Blocker'},
+                                     issuetype={'name': 'Bug'},
+                                     labels=format_labels({"test_id": test_id}))
+
+    logger.info("issue created: %s", new_issue)
+    add_watchers(jclient, new_issue)
+    return new_issue
+
+
+def get_failed_tests():
+    try:
+        res = requests.get(TEST_STATUS_URL)
+        res.raise_for_status()
+        tests = {n:data for n,data in res.json().items() if data['overall_status'] == "FAILING"}
+        return tests
+    except:
+        logger.exception("Error getting list of failed tests")
+        raise
+
+
+def main(arg):
+    if arg.user_password is None:
+        username, password = get_credentials_from_netrc(urlparse(JIRA_SERVER).hostname, arg.netrc)
+    else:
+        try:
+            [username, password] = arg.user_password.split(":", 1)
+        except Exception:
+            logger.error("Failed to parse user:password")
+
+    jclient = get_jira_client(username, password)
+
+    try:
+        failed_tests = get_failed_tests()
+    except Exception:
+        sys.exit(1)
+
+    existing_tickets = get_all_triage_tickets(jclient)
+
+    for failed_test in failed_tests:
+        create_jira_ticket(jclient, existing_tickets, failed_test, failed_tests[failed_test])
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    loginGroup = parser.add_argument_group(title="login options")
+    loginArgs = loginGroup.add_mutually_exclusive_group()
+    loginArgs.add_argument("--netrc", default="~/.netrc", required=False, help="netrc file")
+    loginArgs.add_argument("-up", "--user-password", required=False,
```

Comment:
maybe rename to ``--auth`` ?

Reply:
This is alligned with the other similar script.
I guess that once we consolidate all the jira related code, we can think on the parameter name